data/blog/bitcoin-grants-july-2023.mdx

```diff
@@ -0,0 +1,265 @@
+are delighted to announce grants for over a dozen projects in the bitcoin &
+lightning ecosystem.
+
+[^fn-hal]: Hal Finney: [_Why remailers..._](https://cypherpunks.venona.com/date/1992/11/msg00108.html) (November 1992)
+
+[^fn-assange]: Julian Assange: [_A Call to Cryptographic Arms_](https://cryptome.org/2012/12/assange-crypto-arms.htm) (October 2012)
+
+The following open-source projects were [selected](https://github.com/OpenSats/website/blob/master/docs/grant-selection.md) to by the
+OpenSats board for funding:
+
+- [Payjoin Dev Kit](https://github.com/payjoin)
+- [Bolt12 for LND](https://github.com/ziggie1984)
+- [Splicing](https://lightningsplice.com/)
+- [Raspiblitz](https://github.com/raspiblitz/raspiblitz)
+- [Labelbase](https://github.com/Labelbase/Labelbase)
+- [BTCPay Server](https://github.com/btcpayserver)
+- [ZeroSync](https://github.com/zerosync/zerosync)
+- [Mutiny Wallet](https://github.com/MutinyWallet)
+- [next-auth Lightning Provider](https://github.com/jowo-io/next-auth-lightning-provider)
+- [Cashu](https://github.com/cashubtc/cashu)
+- [lnproxy](https://github.com/lnproxy/lnproxy)
+- [Blixt Wallet](https://github.com/hsjoberg/blixt-wallet)
+
+In addition to the open-source projects listed above, three
+educational initiatives were selected:
+
+- [Bitcoin Education in Nigeria](https://geyser.fund/project/bitcoineducationinnigeriawithapatajohnson)
+- [21 Ideas](https://www.21ideas.org/)
+- [CoreDev.tech](https://coredev.tech/)
+
+The first is an initiative started and led by Apata Johnson. Apata's project
+aims to educate youths on bitcoin and the opportunities it brings for the people
+living in the rural areas of Nigeria.
+
+The second, 21 Ideas, is a project that aims to
+bring quality Bitcoin education to Russian citizens. Tony and others have been
+working for many years on translations, original material, and hands-on
+tutorials for beginners. We believe that education is paramount to proper
+Bitcoin use, and localization is paramount for everyday citizens to properly
+grasp the importance as well as the novel concepts of bitcoin.
+
+Last but not least, we provide financial support for one of the upcoming
+CoreDev developer events, which is all about bringing
+developers together so that they can hack on Bitcoin Core and related software.
+
+With that out of the way, let's take a closer look at the free and open-source
+software projects of this funding round.
+
+---
+
+### Payjoin Dev Kit
+
+Payjoin brings privacy to bitcoin without changing the way you're used to using
+it. Payjoin transactions look no different from normal activity on-chain, so
+they boost everyone's privacy, even those who don't payjoin, and foil chain
+surveillance.
+
+Payjoin is easy to integrate and falls back to working defaults where it isn't
+supported, but it can only take off when senders and receivers include standard
+payjoin support in their software. Payjoin Dev Kit makes it easy to integrate
```

Comment:
Makes it easy for wallet developers(?). As a non-developer I'm unclear on why this project matters (other than knowing that privacy is important).